content/en/data_jobs/dataproc.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Enable Data Jobs Monitoring for Spark on Dataproc - Google Cloud
+further_reading:
+    - link: '/data_jobs'
+      tag: 'Documentation'
+      text: 'Data Jobs Monitoring'
+---
+
+[Data Jobs Monitoring][9] gives visibility into the performance and reliability of Apache Spark applications on Google Cloud Dataproc.
+
+## Requirements
+
+[Dataproc Release 2.0.x][10] or later is required. All of Debian, Rocky Linux, and Ubuntu Dataproc standard images are supported.
+
+
+## Setup
+
+Follow these steps to enable Data Jobs Monitoring for GCP Dataproc.
+
+1. [Store your Datadog API key](#store-your-datadog-api-key-in-google-cloud-secret-manager-recommended) in GCP Secret Manager (Recommended).
+1. [Create and configure your Dataproc cluster](#create-and-configure-your-dataproc-cluster).
+1. [Specify service tagging per Spark application](#specify-service-tagging-per-spark-application).
+
+### Store your Datadog API key in Google Cloud Secret Manager (Recommended)
+1. Take note of your [Datadog API key][1].
+1. In [GCP Secret Manager][2], choose **Create secret**.
+   - Under **Name**, enter a **Secret name**. You can use `datadog_dd_api_key`.
```

Comment:
nit: `dd_api_key` name should be enough (to not repeat datadog), but it means screenshot would have to be changed